next/components/forms/GeneratedFormRJSF.tsx

```diff
@@ -57,6 +57,7 @@ const GeneratedFormRJSF = ({
           'flex flex-col gap-10 pt-0 pb-6 lg:py-10 w-full max-w-screen-lg mx-auto',
           'lg:flex-row lg:gap-20',
           wrapperClassName,
+          {"overflow-hidden fixed": formModalsRef.current?.isAnyModalOpen}
```

Comment:
This should be handled by modal library we use (if we use one). You cannot use ref as it won't trigger rerender.

Reply:
Ok, the current modal solution sucks, we need to migrate to https://react-spectrum.adobe.com/react-aria/Modal.html#modal

Reply:
I create an issue https://github.com/bratislava/konto.bratislava.sk/issues/314

Reply:
Yup, I know. My solution was before modals were extracted. And then when I merged master to my branch, I didn't realize the new component was using refs. I guess this PR can be closed since we need to migrate to a new one (cc @mpinter)?

Reply:
I think I was able to find a way to solve this issue.

Reply:
@JacobMu indeed, let's close this one if we're moving forward with replacing modals completely, the task / solution will likely be obsolete after such replace

@MarekBodingerBA thanks for raising the modal issue 👍